src/main/app/case/CaseApi.ts

```diff
@@ -64,8 +64,7 @@ export class CaseApi {
         throw new Error('Error in updating case, case id is missing');
       }
       const url: string = config.get(FIS_COS_API_BASE_URL);
-      const AdditionalDocuments = req.session['AddtionalCaseDocuments'].map(document => {
-        // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
Remove commented code please

Reply:
@muradalamv1 These are the lines that have been deleted - so that line has been removed. :-)